question-service/model/question-model.js

```diff
@@ -15,7 +15,7 @@ function createQuestionModel(s = sequelize) {
         allowNull: false,
       },
       difficulty: {
-        type: DataTypes.ENUM("EASY", "MEDIUM", "HARD"),
+        type: DataTypes.ENUM("EASY", "MEDIUM", "HARD", "ANY"),
```

Comment:
I think don't need to add try `ANY` for the Question Model

Reply:
You're right, thanks :)